components/create_comment/create_comment.tsx

```diff
@@ -1071,7 +1070,7 @@ class CreateComment extends React.PureComponent<Props, State> {
     }
 
     focusTextbox = (keepFocus = false) => {
-        if (this.textboxRef.current && (keepFocus || !UserAgent.isMobile())) {
+        if (this.textboxRef.current && keepFocus) {
```

Comment:
@ismaaylSpiria Could you please change it here https://github.com/mattermost/mattermost-webapp/blob/ed1e261c5044c935204a2bc4ec2dcdc9dee10e7a/components/advanced_create_comment/advanced_create_comment.tsx#L1046 as well, we recently introduced Advanced Text editor due to which there is some duplication of code, which we will be removed in future.

Reply:
Hi sorry for the delay, 
what should I do to avoid the duplication please? I'm not familliar with the Advanced Text editor

Reply:
@ismaaylSpiria I think I wasn’t clear Enough I meant if you can add your changes here as 
well
https://github.com/mattermost/mattermost-webapp/blob/ed1e261c5044c935204a2bc4ec2dcdc9dee10e7a/components/advanced_create_comment/advanced_create_comment.tsx#L1046

Reply:
oh, yes sure. I'll do that.
thanks

Reply:
this PR seems closed, should I create a new one?

Reply:
@ismaaylSpiria are you able to contribute to it, if not please go ahead and create a new PR